jest.config.js

```diff
@@ -7,4 +7,8 @@ module.exports = {
     "^.+.(css|styl|less|sass|scss|png|jpg|ttf|woff|woff2)$":
       "jest-transform-stub",
   },
+  setupFiles: [
+    "jest-webextension-mock"
```

Comment:
Needed to handle [`webextension-polyfill`](https://github.com/getAlby/lightning-browser-extension/blob/master/src/common/lib/msg.ts#L1)